plugins/vendor/zoom/services/ZoomVendorService.php

```diff
@@ -262,4 +267,36 @@ public function recordingCompleteAction()
 		$this->setPartnerFilters($zoomIntegration->getPartnerId());
 		$kZoomEventHandler->processEvent($event);
 	}
+
+	/**
+	 * Retrieve zoom integration setting object by partner id
+	 *
+	 * @action get
+	 * @param int $partnerId
+	 * @return KalturaZoomIntegrationSetting
+	 * @throws APIErrors::INVALID_PARTNER_ID
+	 */
+	public function getAction($partnerId)
+	{
+		/** @var ZoomVendorIntegration $zoomIntegration */
+		$zoomIntegration = VendorIntegrationPeer::retrieveSingleVendorByPartner($partnerId, VendorTypeEnum::ZOOM_ACCOUNT);
+		if($zoomIntegration)
+		{
+			$integrationSetting = new KalturaZoomIntegrationSetting();
```

Comment:
Instead, you shoudl set mapping between objects of KalturaZoomIntegrationSetting and ZoomVendorIntegration
and function fromObject will this mapping for you.,